### PR TITLE
Add cfn-signal utility.

### PR DESCRIPTION
**Pull request by obmarg**
See README for a description.

### CODE REVIEW

File: cfn-signal/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+eval `ec2cluster`;
```
Comment (lukasz-madon) [inline cfn-signal/run.sh:3]: that eval here is useless

Comment (obmarg) [reply cfn-signal/run.sh:3]: What makes you say that?

Comment (lukasz-madon) [reply cfn-signal/run.sh:3]: because it does nothing. The result of evaluation is `ec2cluster`. Unless I'm missing something.

Comment (obmarg) [reply cfn-signal/run.sh:3]: I think you're missing something.  ec2cluster outputs a bunch of data like:

```
REGION=us-west-1
TAG_AWS_CLOUDFORMATION_STACK_ID=something_or_other
```

We need to put those variables into our environment by calling `eval`.  If I removed `eval`, calling ec2cluster would be absolutely pointless.

**Comment by lukasz-madon** [comment]
👍

**Pull request merged**